src/lib/graphql.js

```diff
@@ -109,13 +109,22 @@ function formatDefaultValue(type, defaultValue) {
 
 function getFilteredTypeMap(
   typeMap,
-  excludeList = SCHEMA_EXCLUDE_LIST_PATTERN,
+  filterOperationKind = ["Query", "Mutation", "Subscription"],
```

Comment:
Do not make this list static, but fetch it from [`getRootType()`](https://github.com/graphql/graphql-js/blob/main/src/type/schema.ts#L276).